x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/components/curations_table.test.tsx

```diff
@@ -0,0 +1,143 @@
+/*
```

Comment:
The contents of this file are unmodified from what used to be in `curations.test.tsx` except for imports